app/src/main/res/drawable-v24/switch_theme_thumb_selector.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
do we need this since we do not have a theme switcher yet ?

Reply:
will remove this in next PR